dev/com.ibm.ws.webcontainer.security/src/com/ibm/ws/webcontainer/security/WebProviderAuthenticatorProxy.java

```diff
@@ -587,18 +587,20 @@ private AuthenticationResult handleOidcClient(HttpServletRequest req, HttpServle
         if (oidcClient == null) {
             return new AuthenticationResult(AuthResult.CONTINUE, "OpenID Connect client is not available, skipping OpenID Connect client...");
         }
+        String provider = oidcClient.getOidcProvider(req);
+        if (provider == null) {
+            return new AuthenticationResult(AuthResult.CONTINUE, "not an OpenID Connect client request, skipping OpenID Connect client...");
+        }
 
         if (firstCall) {
+            oidcClient.logoutIfSessionInactive(req, provider);
```

Comment:
Please add a beta guard. Thanks!

Reply:
beta guard added to `Jose4jUtil` and `OidcClientImpl`